filesystem/gpt_partition_table.ksy

```diff
@@ -0,0 +1,76 @@
+meta:
+  id: gpt_partition_table
+  title: GPT (GUID) partition table
+  endian: le
+  license: CC0-1.0
+doc-ref: Specification taken from https://en.wikipedia.org/wiki/GUID_Partition_Table
+instances:
+  sector_size:
+    value: 0x200
```

Comment:
@GreyCat, is it possible to make it a parameter?

Reply:
Technically, it is possible to have `params` in top-level type. However, this will make it unusable for current visualization tools — AFAIK, none of them allows to visualize parameterized type.